packages/main-menu/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@lidofinance/main-menu",
+  "version": "0.0.0-semantic-release",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
```

Comment:
Just curious, why types are in esm?

Reply:
I don't know, maybe because esm is a more strict module system

Reply:
As far as I know, it's not related 🤔 Will check it a bit later